egui/src/widgets/plot/mod.rs

```diff
@@ -108,6 +110,35 @@ impl LinkedAxisGroup {
     }
 }
 
+#[derive(Debug, Clone, Copy, PartialEq)]
+pub enum HoverLine {
+    None,
+    X,
+    Y,
+    XY,
+}
```

Comment:
Seems like it would make more sense to make this into a

```
pub struct HoverLine {
    /// Highlight values on the X axis by painting a vertical line.
    pub x: bool,
    /// Highlight values on the Y axis by painting a horizontal line.
    pub y: bool
}
```